src/Features/Core/Portable/Completion/AbstractCompletionService.cs

```diff
@@ -277,12 +278,29 @@ private static CompletionItem Disambiguate(CompletionItem item, CompletionItem e
             CancellationToken cancellationToken)
         {
             var context = new CompletionListContext(document, position, triggerInfo, options, cancellationToken);
-
-            await provider.ProduceCompletionListAsync(context).ConfigureAwait(false);
+            var root = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
Some comment to some tracking bug to follow up on this would be great. i.e. to check for telemetry, delete the check if we figure it out, etc.